frontend/src/lib/constants.tsx

```diff
@@ -122,12 +122,10 @@ export const FEATURE_FLAGS = {
     KAFKA_INSPECTOR: 'kafka-inspector', // owner: @yakkomajuri
     INSIGHT_EDITOR_PANELS: '8929-insight-editor-panels', // owner: @mariusandra
     FRONTEND_APPS: '9618-frontend-apps', // owner: @mariusandra
-    TOOLBAR_LAUNCH_SIDE_ACTION: 'toolbar-launch-side-action', // owner: @pauldambra,
```

Comment:
Is this still needed then: https://github.com/PostHog/posthog/blob/0c1c3696122f1fc71e0b91a534799d3db08325ee/cypress/support/e2e.js#L22?